admin_guide/install/advanced_install.adoc

```diff
@@ -137,42 +160,202 @@ master.example.com openshift_node_labels="{'region': 'infra', 'zone': 'default'}
 node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
 node2.example.com openshift_node_labels="{'region': 'primary', 'zone': 'west'}"
 ----
-endif::[]
-ifdef::openshift-origin[]
+====
+
+Modify the file to match your environment and specifications, and save it as
+*_/etc/ansible/hosts_*.
+
+[[single-master-multi-etcd-multi-node]]
+
+=== Single Master, Multiple etcd, and Multiple Nodes
+
+The following table describes an example environment for a single
+link:../../architecture/infrastructure_components/kubernetes_infrastructure.html#master[master],
+three
+link:../../architecture/infrastructure_components/kubernetes_infrastructure.html#master[*etcd*]
+hosts, and two
+link:../../architecture/infrastructure_components/kubernetes_infrastructure.html#node[nodes]:
+
+[options="header"]
+|===
+
+|Host Name |Infrastructure Component to Install
+
+|*master.example.com*
+|Master and node
+
+|*etcd1.example.com*
+.3+.^|etcd
+
+|*etcd2.example.com*
+
+|*etcd3.example.com*
+
+|*node1.example.com*
+.2+.^|Node
+
+|*node2.example.com*
+|===
+
+[NOTE]
+====
+When specifying multiple *etcd* hosts, external *etcd* is installed and
+configured. Clustering of OpenShift's embedded *etcd* is not supported.
```

Comment:
@detiber I don't disagree. All the bolding was just me trying to follow our markup guidelines. etcd seemed to fall under the bold-ish option:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference